DS/DS6/EmployeeSortingByName.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+public class EmployeeSortingByName implements Comparable<Object>{
+	String employeeID,nameOfEmployee,addressOfEmployee;
+	public EmployeeSortingByName(String employeeID,String nameOfEmployee,String addressOfEmployee) {
+		this.employeeID=employeeID;
+		this.nameOfEmployee=nameOfEmployee;
+		this.addressOfEmployee=addressOfEmployee;
+	}
+	public String getNameOfEmployee() {
```

Comment:
Why do you have to create a function to get employee name, This function is used only inside this class. We can directly access the variable of this class within other functions.